style.css

```diff
@@ -1 +1,322 @@
-/*  Add your styling here */
+body {
+    margin: 0;
+    padding: 0;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+/* Header Styles */
+
+header {
+    margin: 0 0 8px 0;
+    padding: 0;
+    width: 100%;
+    background-color: #F3CFC6;
+    border-bottom: 1px solid #cac5c5;
+}
+
+.nav-menu {
+    display: none;
+    padding: 0;
+}
+
+.nav-ul {
+    list-style: none;
+    display: none;
+    flex-direction: row;
+    gap: 10px;
+}
+
+.nav-list {
+    display: none;
+}
+
+.nav-item {
+    padding: 0;
+    font-size: 1em;
+    text-transform: uppercase;
+    color: #2c2828;
+    font-weight: bold;
+    text-decoration: none;
+    display: none;
+}
```

Comment:
There is no need to have display:none; on all of these elements if you have it on `.nav-menu` it will hide all its children then if you add
```
.nav-menu.active {
  display: block;
}
```
it should all work

Reply:
I understand.